tests/test_pyxis.py

```diff
@@ -598,3 +602,70 @@ def test_is_bundle_false(self, page):
+                               '_id': '087721b',
+                               'registry': 'registry.access.redhat.com',
+                               'repository': 'rhel8/thunderbird-flatpak',
+                               'tags': [{'_links': {}, 'name': 'flatpak-8040020210719090808.1'}],
+                               'published': True
+                               },
+                              {'_links': {},
+                               '_id': '087721c',
+                               'registry': 'registry-proxy.engineering.redhat.com',
+                               'repository': 'rh-osbs/thunderbird',
+                               'tags': [{'_links': {}, 'name': 'flatpak-8040020210719090808.1'},
+                                        {'_links': {}, 'name': 'rhel-8.4.0-z-containers-candidate-35976-20210719181035-s390x'}],
+                               'published': False
+                               }
+                              ]
+             }]
+        metadata = self.px.get_image_metadata_by_nvr('thunderbird-flatpak-container-flatpak-8040020210719090808.1')
+        expected_data = {'_id': '087721b',
+                         'registry': 'registry.access.redhat.com',
+                         'repository': 'rhel8/thunderbird-flatpak',
+                         'tags': ['flatpak-8040020210719090808.1']
+                         }
+        self.assertDictEqual(metadata, expected_data)
+
+    @patch('freshmaker.pyxis.Pyxis.get_auto_rebuild_tags')
+    @patch('freshmaker.pyxis.Pyxis._pagination')
+    def test_rebuild_image(self, page, get_auto_rebuild_tags):
+        page.return_value = [
+            {'_links': {},
+             'repositories': [{'_links': {},
+                               '_id': '087721b',
+                               'published': True,
+                               'registry': 'registry.access.redhat.com',
+                               'repository': 'rhel8/thunderbird-flatpak',
+                               'tags': [{'_links': {}, 'name': 'flatpak-8040020210719090808.1'}],
+                               'published': True
+                               }]
+             }]
+        get_auto_rebuild_tags.return_value = ['flatpak-8040020210719090808.1',
+                                              'latest'
+                                              ]
+        rebuild_image_id = self.px.get_rebuild_image_id('thunderbird-flatpak-container-flatpak-8040020210719090808.1')
+        self.assertEqual(rebuild_image_id, '087721b')
+
+    @patch('freshmaker.pyxis.Pyxis.get_auto_rebuild_tags')
+    @patch('freshmaker.pyxis.Pyxis._pagination')
+    def test_non_rebuild_image(self, page, get_auto_rebuild_tags):
+        page.return_value = [
+            {'_links': {},
+             'repositories': [{'_links': {},
+                               '_id': '087721b',
+                               'published': True,
+                               'registry': 'registry.access.redhat.com',
```

Comment:
Could you change this to some fake string? This is public/open source, so maybe let's not mention red hat.

Reply:
Ack. Fixing